src/v2/view-builder/internals/FormInputFactory.js

```diff
@@ -102,6 +118,10 @@ const create = function(uiSchemaObj) {
  *
  */
 const createIdpButtons = (remediations) => {
+  // This is the max allowable text width that we can display for IDP buttons.
+  // Generated by taking the 316px width of button minus padding-left (90 px) and padding-right (50px)
+  const MAX_WIDTH = 186;
```

Comment:
Can we handle this via CSS instead?

Reply:
@edburyenegren-okta this is strictly for doing the calculation on the fly of whether the IDP buttons we render will overflow. Although not ideal, we need the max width in here to compare against, I don't see a way to determine that part via CSS. We ultimately use this to determine whether we add a `title` attr. We still handle the actual truncating and displaying of the ellipsis via CSS.